src/renderer/src/App.css

```
@@ -581,6 +581,25 @@ textarea {
   animation: fadeSlide 0.25s ease-out backwards;
 }
 
+/* The mirror of fadeSlide, for a surface that floating-ui has flipped ABOVE its
+   anchor (the profile menu on the bottom row, #884): it should slide up toward
+   the anchor, not down away from it. */
+@keyframes fadeSlideUp {
```

Stylelint (17.14.0)

[error] Expected keyframe name "fadeSlideUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)